packages/@aws-cdk/custom-resources/lib/provider-framework/provider.ts

```diff
@@ -178,7 +180,7 @@ export class Provider extends CoreConstruct implements cfn.ICustomResourceProvid
    * Called by `CustomResource` which uses this provider.
    * @deprecated use `provider.serviceToken` instead
    */
-  public bind(_: CoreConstruct): cfn.CustomResourceProviderConfig {
+  public bind(_: CoreConstruct): CustomResourceProviderConfig {
```

Comment:
There's a bug in go that does not allow params named `_` so it would be nice to change this if you can:

```suggestion
  public bind(_scope: CoreConstruct): CustomResourceProviderConfig {
```